clients/drcachesim/common/options.cpp

```diff
@@ -395,7 +395,9 @@ droption_t<std::string> op_record_heap_value(
     OP_RECORD_FUNC_ITEM_SEP,
     "malloc|0|1" OP_RECORD_FUNC_ITEM_SEP "free|1|1" OP_RECORD_FUNC_ITEM_SEP
     "tc_malloc|2|1" OP_RECORD_FUNC_ITEM_SEP "tc_free|3|1" OP_RECORD_FUNC_ITEM_SEP
-    "__libc_malloc|4|1" OP_RECORD_FUNC_ITEM_SEP "__libc_free|5|1",
+    "__libc_malloc|4|1" OP_RECORD_FUNC_ITEM_SEP "__libc_free|5|1" OP_RECORD_FUNC_ITEM_SEP
+    "operator new|10|1" OP_RECORD_FUNC_ITEM_SEP
+    "operator delete|11|1" OP_RECORD_FUNC_ITEM_SEP "calloc|6|2",
```

Comment:
It would be nice to have these additions show up in a test: the fact that this PR needs zero changes to any tests is not ideal.  Would it be straightforward to either add a test or add checking to an existing test that exercises this option change here?

Reply:
Done.